packages/editor/src/components/post-text-editor/index.js

```diff
@@ -113,14 +112,13 @@ export default compose( [
 		};
 	} ),
 	withDispatch( ( dispatch ) => {
-		const { editPost, resetBlocks, checkTemplateValidity } = dispatch( 'core/editor' );
+		const { editPost } = dispatch( 'core/editor' );
 		return {
 			onChange( content ) {
-				editPost( { content } );
+				editPost( { content }, { skipContentParse: true } );
```

Comment:
This makes me think, this should be a local state and only called when `onPersist` is called.

Reply:
Perhaps.

Good: It might help us avoid the need for this `skipContentParse` flag altogether.
Bad: We rely on `editPost` to trigger the change detection which would alert the user to their changes being unsaved. As noted in the original comment, this isn't entirely perfect as implemented anyways.